projects/packages/connection/src/class-manager.php

```diff
@@ -111,7 +111,6 @@ public static function configure() {
 		);
 
 		$manager->setup_xmlrpc_handlers(
-			$_GET, // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I personally think it's best to avoid passing the whole `$_GET` here. Actually it made more sense to me to directly use only the request params we need inside `setup_xmlrpc_handlers` as this doesn't seem something that the method consumers need to be aware of.

I've audited the usage of `setup_xmlrpc_handlers` and it's only used by the same class. It could even be a private or protected method to indicate that but I'm not sure why it was made a public one in the first place.

Reply:
Thanks! I guess we can keep it public _for now_ so that we don't have to refactor the unit tests and update the changelog via https://github.com/Automattic/jetpack/pull/37392/commits/43180fc8cc9491831dfc232a74ca9740ca1d437d